README.md

```diff
@@ -42,31 +49,32 @@ repositories:
     visibility: public | private # Default is public
 ```
 
-Note: the ```name: repo name``` does not necessarily appear as the first field in a repositories entry which can be confusing.
+Note: the ```name: REPO_NAME``` does not necessarily appear as the first field in a repositories entry which can be confusing.
```

Comment:
Maybe it'd be worth to fix the configuration file to make the process easier for users. I guess it was generated by Sheriff, hence the unexpected fields order.

Reply:
You are correct to say that this issue predates the CLOWarden implementation. I'm not 100% sure who or what did the alphabetic sort (It may or may not have been Sheriff itself, no matter what way the keys are sorted the YAML is valid and consumable.)

Against adding new code for this is the fact that the CW Validation is so good! This is not as serious an issue because the validation step makes clear if there any problems.

If you want to do something on this the only word of caution I would raise is the size of the config file over on the CNCF org; rolling out an org-wide change there might be disruptive, incremental repo changes one PR at a time less so; Smaller orgs like cncf-tags would be a good place to test out such changes if you wanted to consider adding configurable capability?

Reply:
Fair enough! :-D I'll encourage people to ensure that the repo name first for ease of use.